recipes/openssl/config.yml

```diff
@@ -1,19 +1,19 @@
 versions:
   # 3.0.x releases
+  3.0.6:
```

Comment:
Is this supposed to be 3.0.7 as per title?

Reply:
Sure, yes🤦‍♂️
Got confused there, because the 3.0.6 got retracted shortly after its release